Sample/Kitchen Sink/DemonstrationTools/DemonstrationViewModel.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  DemonstrationViewModel.swift
+//  Kitchen Sink
+//
+//  Created by Jake Dove on 11/03/2024.
+//
```

Comment:
```
//
//  DemonstrationViewModel.swift
//  Flight UI - Kitchen Sink Sample
//
//  Created by Appivate 2023
//

```

Not sure about this one, but the comments at the top of the files don't include developer name in other files 🤔  This may be an established convention or it may not 🤷🏻